### ISSUE TITLE
Public key for signing releases missing

### ISSUE THREAD

**Issue by Unit193** [obynio/certbot-plugin-gandi#28]
Howdy,

First off, thanks for maintaining certbot-plugin-gandi.  I noticed a while back you seem to have changed the signing key for the releases on pypi, but there wasn't any notice of change and the public key doesn't appear to be online anywhere.

The last release was signed by 86866EAF84F46D74E263C078D80EC4E1F40970E5, and this is currently what's holding up Debian for updating.

~Unit 193

**Comment by obynio** [comment]
Hello, pretty late to the party

>  I noticed a while back you seem to have changed the signing key for the releases on pypi, but there wasn't any notice of change and the public key doesn't appear to be online anywhere.

I'm working to fix that. PyPI has unfortunately deprecated signing package. I recently sent a mail to the team maintaining the debian packages but never received an answer. Do you know how I can reach them ?

**Comment by Unit193** [comment]
Yeah at this point it's likely moot.  Well, I'm the person that initially packaged it and am still listed as a maintainer, but I recently moved everything I had to Porkbun too.. I was considering filing to get it removed from Debian, but honestly it should just be taken over as it's still quite useful to someone using Gandi.

**Comment by obynio** [comment]
Would it be possible to take over the package maintenance ? I keep receiving messages about this package because it's not working anymore because the API changed on Gandi's side.

**Comment by Unit193** [comment]
Sorry for the delay, but yeah I wouldn't see why not?  It hasn't been a priority as a RC bug is keeping it out of stable anyway, but this doesn't help Ubuntu users.  The current RC bug relates to a broken [upgrade path](https://bugs.debian.org/1038920).  I have considered filing a bug to remove the plugin altogether, even though I still think it's useful for Gandi users.  I'm not sure which path would be most useful to you?

**Comment by obynio** [comment]
Hello, the new argument they talk about is not my doing, it's certbot that changed the argument in their side unilaterally. I would consider removing the plugin since the plugin version is too old and does not work anymore due to recent API changes on [Gandi's side](https://api.gandi.net/docs/authentication/).

**Comment by obynio** [comment]
Okay @Unit193 I'm not very familiar with Debian build system, what can I do to make sure this package is upgraded on the debian repositories ?
